pkg/apis/kf/v1alpha1/build_lifecycle.go

```diff
@@ -64,7 +59,7 @@ func (status *BuildStatus) PropagateBuildStatus(build *build.TaskRun) {
 
 	cond := build.Status.GetCondition(apis.ConditionSucceeded)
 	if PropagateCondition(status.manage(), BuildConditionTaskRunReady, cond) {
-		status.Image = GetTaskRunOutputResource(build, TaskRunResourceNameImage, TaskRunResourceURL)
```

Comment:
Should we keep both but prefer the newer one over the older so if the upgrade happens while Tekton is still processing a job the value will come through?